src/ServiceControl.Persistence.RavenDB/SagaAudit/NoOpSagaAuditDataStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ServiceControl.Persistence.RavenDB
+{
+    using System;
+    using System.Threading.Tasks;
+    using ServiceControl.Persistence.Infrastructure;
+    using ServiceControl.SagaAudit;
+
+    class NoOpSagaAuditDataStore : ISagaAuditDataStore
+    {
+        public Task<bool> StoreSnapshot(SagaSnapshot sagaSnapshot) => Task.FromResult(false);
+
+        public Task<QueryResult<SagaHistory>> GetSagaById(Guid sagaId) => Task.FromResult(QueryResult<SagaHistory>.Empty());
```

Comment:
Technically, this task could be cached but given this is not really on the critical path I'd say leave it as is

Reply:
No longer relevant with #4175